models/demos/falcon7b/tt/model_config.py

```diff
@@ -129,6 +130,55 @@ def get_model_config(model_config_str, prefill_seq_len=0):
     model_config.update({f"{key}_MEMCFG": mem_config for key in OP_KEYS if key not in NO_MEMCFG})
     model_config.update({f"{key}_DTYPE": dtype for key in OP_KEYS if key not in NO_DTYPE})
 
+    ln_max_num_cores_y = 8
```

Comment:
we should not use prefill_seq_len in model config, this should be part of model init

Reply:
ideally it shouldn't be a part of model init as well.. both model config and model initialization should be agnostic to seq len; These seqlen-related calculations should go either in forward pass or model preprocessing.

I have to say though I see now both Falcon40b and Llama use seq_len in the model config construction so maybe my strong feelings about model construction/config being agnostic to seq_len can be challenged